packages/contracts-bedrock/src/interop/InteropL2StandardBridge.sol

```diff
@@ -0,0 +1,203 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { L2StandardBridge as LegacyL2StandardBridge } from "src/L2/L2StandardBridge.sol";
+import { InteropL2CrossDomainMessenger } from "src/interop/InteropL2CrossDomainMessenger.sol";
+import { InteropConstants } from "src/interop/InteropConstants.sol";
+
+import { IOptimismMintableERC20 } from "src/universal/IOptimismMintableERC20.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { SafeCall } from "src/libraries/SafeCall.sol";
+import { ISemver } from "src/universal/ISemver.sol";
+
+import { ERC165Checker } from "@openzeppelin/contracts/utils/introspection/ERC165Checker.sol";
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+
+/// @title InteropL2StandardBridge
+/// @notice InteropL2StandardBridge is a replacement of the L2StandardBridge that supports
+///         both L2-L2 and L2-L1 bridging.
+contract InteropL2StandardBridge is ISemver {
```

Comment:
nit: naming is verbose

Reply:
yea not a fan of the name. Originally had it just `L2StandardBridge` & `NewL2StandardBridge` but the name collisions with forge and the bindings made that not work